client/src/components/InternshipCompanyInfo.jsx

```diff
@@ -1,7 +1,34 @@
-import { Avatar, Box, Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
 import React from 'react';
 
+import amazonLogo from '../assets/company_logos/amazon_logo.svg';
+import appleLogo from '../assets/company_logos/apple_logo.svg';
+import genericLogo from '../assets/company_logos/generic.svg';
+import googleLogo from '../assets/company_logos/google_logo.svg';
+import metaLogo from '../assets/company_logos/meta_logo.svg';
+import microsoftLogo from '../assets/company_logos/microsoft_logo.svg';
+import netflixLogo from '../assets/company_logos/netflix_logo.svg';
+
 const InternshipCompanyInfo = (props) => {
+  const getCompanyIcon = (company) => {
+    switch (company) {
```

Comment:
can we make this case insensitive in case there is a misspelling? ideally, we have enums for companies in the future => will also depend on the backend